scripts/form-utils.js

```diff
@@ -114,22 +103,17 @@ function postToEmailServer(params, successCallback, errorCallback) {
     xhr.timeout = 5000;
     xhr.ontimeout = errorCallback;
     xhr.responseType = 'json';
-
```

Comment:
There is no `response.desc` so we can just drop that and call `errorCallback` without a message. Just throw the error if `response.status` is "error".